src/main/java/com/example/bookstore/model/Author.java

```diff
@@ -1,4 +1,57 @@
 package com.example.bookstore.model;
 
-public class Author {
+public class Author{
+    // Поля класу; class attributes
+    private String firstName;
+    private String lastName;
+    private String birthDate;
```

Comment:
You did a great job! But I have one small remark about birthDate field. Can you replace it with LocalDate class?